conf/otherBeneficiary.routes

```diff
@@ -19,9 +19,17 @@ POST        /:draftId/:index/uk-address                 controllers.register.ben
 GET         /:draftId/:index/non-uk-address             controllers.register.beneficiaries.other.NonUkAddressController.onPageLoad(index: Int, draftId: String)
 POST        /:draftId/:index/non-uk-address             controllers.register.beneficiaries.other.NonUkAddressController.onSubmit(index: Int, draftId: String)
 
-
 GET         /:draftId/:index/check-details              controllers.register.beneficiaries.other.CheckDetailsController.onPageLoad(index: Int, draftId: String)
 POST        /:draftId/:index/check-details              controllers.register.beneficiaries.other.CheckDetailsController.onSubmit(index: Int, draftId: String)
 
 GET        /:draftId/:index/remove                     controllers.register.beneficiaries.other.RemoveOtherBeneficiaryController.onPageLoad(index: Int, draftId: String)
 POST       /:draftId/:index/remove                     controllers.register.beneficiaries.other.RemoveOtherBeneficiaryController.onSubmit(index: Int, draftId: String)
+
+GET         /:draftId/:index/uk-resident               controllers.register.beneficiaries.other.mld5.UKResidentYesNoController.onPageLoad(index: Int, draftId: String)
+POST        /:draftId/:index/uk-resident               controllers.register.beneficiaries.other.mld5.UKResidentYesNoController.onSubmit(index: Int, draftId: String)
+
+GET         /:draftId/:index/has-country-of-residence            controllers.register.beneficiaries.other.mld5.CountryOfResidenceYesNoController.onPageLoad(index: Int, draftId: String)
```

Comment:
Very picky but not overly keen on `has-country-of-residence`. If that's how it's been done for the other types of beneficiary then fair enough for consistency's sake, but I think `know-country-of-residence` or `country-of-residence-yes-no` would be better. I'll leave it up to you.

Reply:
Spoke to Jack about this, leaving at the moment just following the pattern the other routes set